samples/caf_sensor_manager/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -18,4 +18,3 @@ CONFIG_CAF_SENSOR_DATA_AGGREGATOR=n
 CONFIG_CAF_SENSOR_DATA_AGGREGATOR_EVENTS=y
 
 CONFIG_SENSOR=n
-CONFIG_SENSOR_SIM=n
```

Comment:
Shouldn't we leave this line? In prj.conf we set CONFIG_SENSOR_SIM=y and in multicore variant of sample we don't need it.

Reply:
Right. This is a leftover from try of dts based configuration. Thank you for this catch.